build-machinery/make/default.mk

```diff
@@ -0,0 +1,18 @@
+self_dir := $(dir $(lastword $(MAKEFILE_LIST)))
+
+# We extend the default verify/update for Golang
+
+verify: verify-codegen
+update: update-codegen
+
+verify: verify-bindata
```

Comment:
having multiple rules per target works? Never seen this before.

Reply:
It works. Commands must  be defined in one place though.